pkg/sfu/downtrack.go

```diff
@@ -1899,26 +1903,28 @@ func (d *DownTrack) retransmitPackets(nacks []uint16) {
 			} else {
 				ddBytes = epm.ddBytes[:epm.ddBytesSize]
 			}
-			pkt.Header.SetExtension(uint8(d.dependencyDescriptorExtID), ddBytes)
+			if len(ddBytes) != 0 {
```

Comment:
Without this check, we were adding a zero-length extension which made the remote end throw away extensions after this.